Refresh.GameServer/Extensions/LevelEnumerableExtensions.cs

```diff
@@ -36,14 +38,17 @@ public static IQueryable<GameLevel> FilterByLevelFilterSettings(this IQueryable<
         if (levelFilterSettings.ExcludeMyLevels && user != null)
             levels = levels.Where(l => l.Publisher != user);
 
-        levels = levelFilterSettings.GameFilterType switch {
-            GameFilterType.LittleBigPlanet1 => levels.Where(l => l._GameVersion == (int)TokenGame.LittleBigPlanet1),
-            GameFilterType.LittleBigPlanet2 => levels.Where(l => l._GameVersion == (int)TokenGame.LittleBigPlanet2),
-            //NOTE: ideally this should be .Where(l => l._GameVersion == (int)TokenGame.LittleBigPlane1 || l._GameVersion == (int)TokenGame.LittleBigPlane2)
-            //      however, there should be no differences in all real-world cases
-            GameFilterType.Both => levels,
-            _ => throw new ArgumentOutOfRangeException(),
-        };
+        if (levelFilterSettings.GameVersion != TokenGame.BetaBuild)
+        {
+            levels = levelFilterSettings.GameFilterType switch {
+                GameFilterType.LittleBigPlanet1 => levels.Where(l => l._GameVersion == (int)TokenGame.LittleBigPlanet1),
+                GameFilterType.LittleBigPlanet2 => levels.Where(l => l._GameVersion == (int)TokenGame.LittleBigPlanet2),
+                //NOTE: ideally this should be .Where(l => l._GameVersion == (int)TokenGame.LittleBigPlane1 || l._GameVersion == (int)TokenGame.LittleBigPlane2)
+                //      however, there should be no differences in all real-world cases
+                GameFilterType.Both => levels,
+                _ => throw new ArgumentOutOfRangeException(),
+            };
+        }
```

Comment:
It should be commented on why this filtering is disabled for beta builds, i was confused when i first read this code